python/rikai/spark/functions/vision.py

```diff
@@ -425,3 +431,28 @@ def spectrogram_image(
         format=image_format,
         **image_kwargs,
     )
+
+
+@udf(returnType=ArrayType(ImageType()))
+def crop(img: Image, boxes: List[Box2d]):
+    """Crop images from the original image
+
+    Parameters
+    ----------
+    img : Image
+        The base image to crop from.
+    boxes : List[Box2d]
+        A list of bounding boxes to specify the area to crop.
```

Comment:
Can you add documentation on whether these bounding boxes are normalized? i.e., should they have range [0, 1] or range [0, image_width]?

Reply:
Good point.

Reply:
Maybe we should just use normalized box across rikai. what do you think @changhiskhan